dashboard/app/models/script_level.rb

```diff
@@ -21,10 +21,13 @@
 #  index_script_levels_on_stage_id   (stage_id)
 #
 
+require 'cdo/shared_constants'
+
 # Joins a Script to a Level
 # A Script has one or more Levels, and a Level can belong to one or more Scripts
 class ScriptLevel < ActiveRecord::Base
   include LevelsHelper
+  include SharedConstants
```

Comment:
That's already what [include](https://ruby-doc.org/core-2.2.0/Module.html#method-i-include) does. It essentially applies constants, methods, etc. from the included module into the calling module or class. 

You can't filter and only include specific constants this way like with javascript `import {variable} from 'file'`, but including the `SharedConstants` module here will apply each member (`LevelKind` and `LevelStatus`) to the outer `ScriptLevel` class.